lib/actions/preload.coffee

```diff
@@ -163,8 +169,7 @@ module.exports =
 		.then (docker) ->
 
 			# Build the preloader image
-			buildOutputStream = preload.build(docker)
-			buildOutputStream.pipe(process.stdout)
```

Comment:
That's usual docker build stuff.
Like `Step 1/7 .....`

I'd remove it completely, @jhermsmeier mentioned in an earlier PR that it might be useful for the caller.
OK for showing it only if DEBUG is enabled.
@pimterry what values are true for `process.env.DEBUG` ? 1, true, True and TRUE or anything as long as it is defined ?